<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilmInfo</class>
 <widget class="QDialog" name="FilmInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1021</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Podaci o filmu</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>645</width>
        <height>228</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>32</pointsize>
      </font>
     </property>
     <property name="text">
      <string>REZERVACIJE</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLabel" name="labelImage">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>250</height>
      </size>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_Name">
         <property name="text">
          <string>Originalni naslov:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_Name_2">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_ProjectionsStart">
         <property name="text">
          <string>Pocetak prikazivanja:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_ProjectionsStart_2">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_Length">
         <property name="text">
          <string>Duzina trajanja:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_Length_2">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_Country">
         <property name="text">
          <string>Drzava:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_Country_2">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_Year">
         <property name="text">
          <string>Godina:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_Year_2">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_Genre">
         <property name="text">
          <string>Zanr:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_Genre_2">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_Actors">
         <property name="text">
          <string>Glumci:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_Actors_2">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_Director">
         <property name="text">
          <string>Reziser:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_Director_2">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_Distributor">
         <property name="text">
          <string>Distributer:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_Distributor_2">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QPushButton" name="pushButton_Back">
     <property name="text">
      <string>Nazad</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
